packages/contractkit/src/utils/celo-tx.ts

```diff
@@ -0,0 +1,7 @@
+import { Tx } from 'web3/eth/types'
+
+export interface CeloTx extends Tx {
```

Comment:
I believe we have this interface in contractkit already

Reply:
I see you deleted it
maybe try to merge with `types/web3.d.ts`

Reply:
Ok I can use the `Tx` from the `web3/eth/types` that's the same

Reply:
Removed